File: MATLAB/FizzBuzzSimple.m
% Simple MATLAB FizzBuzz
% Author: @veotani
for number = 1:101
    if  mod(number, 15) == 0
        fprintf('FizzBuzz')
    elseif  mod(number, 3) == 0
        fprintf('Fizz')
    elseif mod(number, 5) == 0
        fprintf('Buzz')
    else 
        fprintf(number)
    end
end

File: Go/Fizzbuzz-simple.go
// Simple fizzbuzz implementation
// Author: @SkyC0der
package main

import "fmt"

func main() {
    for i := 1; i <= 100; i++ {
        s := ""       
        if i % 3 == 0 {
            s += "Fizz"
        }
        if i % 5 == 0 {
            s += "Buzz"
        }
        if s != "" {
            fmt.Println(s) 
        } else {
            fmt.Println(i) 
        }
    }
}